.github/actions/browser/action.yml

```diff
@@ -0,0 +1,82 @@
+name: 'Run Test On Browser'
+description: 'Setup Browser and Run Test'
+inputs:
+  # Support one browser per action for now, because some packages has build script before test
+  # which nx cannot send params to karma test after build
+  browsers:
+    description: 'Browser name'
+    required: false
+    default: 'default'
+  mode:
+    description: 'Test mode'
```

Comment:
Better description would be 'Run test in all packages or only affected packages'

Reply:
Fixed